docs/developers/protocol/hermez-protocol/protocol.md

```diff
@@ -526,7 +526,7 @@ Account could be created for a given:
 ### L2
 All L2 transactions are sent to the coordinators by the users. The coordinator collects them into a batch in order to forge it. 
 The coordinator must check that it collects valid transactions that must not perform an invalid transition state. Otherwise, the proof computed by the coordinator will not be valid.
-The user could submit any transaction data to the coordinator but it will be rejected if the transaction could not be processed. Therefore, it is on the user benefit to provide valid transaction if they want it to be inserted in the zkRollup.
+The user could submit any transaction data to the coordinator but it will be rejected if the transaction could not be processed. Therefore, it is in the users benefit to provide a valid transaction if they want it to be inserted in the zkRollup.
```

Comment:
users' benefit?

Reply:
Yes, that is correct. I missed that one, thanks!